packages/primitives/src/Box/Box.tsx

```diff
@@ -101,7 +111,12 @@ type BoxProps = {
   BackgroundProps &
   BorderProps &
   PositionProps &
-  ShadowProps &
+  ShadowProps
```

Comment:
wonder if we should also plan to include some `shadows` as a part of the theme as well similar to how tailwind has some default shadows https://tailwindcss.com/docs/box-shadow.

Reply:
Absolutely, we can start building out the Jilt theme so that we can easily port it over when we start using these components.